content/en/blog/2022/gateway-api-beta/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Extending Gateway API support in Istio"
+publishdate: 2022-07-13
+attribution: "Craig Box (Google)"
+keywords: [traffic-management,gateway,gateway-api,api,gamma,sig-network]
+---
+
+Today we want to [congratulate the Kubernetes SIG Network community on the beta release of the Gateway API specification](https://kubernetes.io/blog/2022/07/12/gateway-api-first-beta/). Alongside this milestone, we are pleased to announce that support for using the Gateway API in Istio ingress is being promoted to Beta, and our intention for the Gateway API to become the default API for all Istio traffic management in the future. We are also excited to welcome our friends from the Service Mesh Interface (SMI) community, who are joining us in a new effort to standardize service mesh use cases using the Gateway API.
```

Comment:
```
Alongside this milestone, we are pleased to announce that support for using the Gateway API in Istio ingress is being promoted to Beta,
```
Starting with 1.15 or effective immediately? Should clarify?

Reply:
This is to be a documentation change, rather than a next release change.  The post is being deliberately ambiguous, as I had wanted to say "today" but it will actually take a few days to fill in the checklist to ensure that this is officially in Beta.

Reply:
I had the same thought as Frank as it was mentioned during the TOC meeting to work on the Enhancements PR (this work would be done `soon`). as an FYI, once that merges and we rerun the ref doc pull, the Feature Status page will get updated. I don't see a PR in istio/enhancements yet.

Reply:
(John suggests "ingress" meets the bar for Beta, but not automated deployment, and definitely not mesh.)